recipe/meta.yaml

```diff
@@ -1,41 +1,79 @@
-{% set version = "0.4.6" %}
+{% set version = "0.5.0" %}
 
 package:
-  name: pyct
+  name: pyct-packages
   version: {{ version }}
 
+source:
+  url: https://pypi.io/packages/source/p/pyct/pyct-{{ version }}.tar.gz
+  sha256: dd9f4ac5cbd8e37c352c04036062d3c5f67efec76d404761ef16b0cbf26aa6a0
+
 build:
-  number: 0
   noarch: python
+  number: 0
 
 requirements:
-  host:
-    - python
-    - pip
-  run:
-    - python
-    - pyct-core {{ version }}
-    - pyyaml
-    - requests
-
-test:
-  imports:
-    - pyct
-    - pyct.cmd
-  requires:
-    - flake8
-    - pytest
+  build:
+    - python >=3.7
+
+outputs:
+  - name: pyct-core
+    build:
+      noarch: python
+      entry_points:
+        - pyct=pyct.__main__:main
+      script: {{ PYTHON }} -m pip install . -vv
+
+    requirements:
+      run_constrained:
+        - pyct {{ version }}
+      host:
+        - python >=3.7
+        - param >=1.7.0
+        - pip
+      run:
+        - python >=3.7
+        - param >=1.7.0
+        - setuptools >=61.0
+        # cli
+        - pyyaml
+        - requests
+
+    test:
+      imports:
+        - pyct
+      commands:
+        - pip check
+        - pyct --help
+      requires:
+        - pip
+
+  - name: pyct
+    build:
+      noarch: python
+
+    requirements:
+      host:
+        - python >=3.7
+        - pip
+      run:
+        - {{ pin_subpackage('pyct-core', exact=True) }}
+
+    test:
+      imports:
+        - pyct
+        - pyct.cmd
 
 about:
-  home: http://github.com/pyviz/pyct
+  home: https://github.com/holoviz-dev/pyct
   license: BSD-3-Clause
   license_family: BSD
-  # in pyct-core
-  # license_file: LICENSE.txt
-  summary: 'Python package common tasks for users (e.g. copy examples, fetch data, ...)'
-  doc_url: https://github.com/pyviz/pyct
-  dev_url: https://github.com/pyviz/pyct
+  license_file: LICENSE.txt
+  summary: Common tasks for package building (e.g. bundle examples)
+  doc_url: https://github.com/holoviz-dev/pyct#pyctbuild
+  dev_url: https://github.com/holoviz-dev/pyct
 
 extra:
   recipe-maintainers:
+    - ceball
```

Comment:
Why are you adding another user? You can add yourself though.

Reply:
Taken from here since this PR essentially merges pyct-core-feedstock into this one: https://github.com/conda-forge/pyct-core-feedstock/blob/main/recipe/meta.yaml#L54